src/Color/Color.js

```diff
@@ -383,40 +261,6 @@ Color.propTypes = {
    * color: This is the selected colour in a rgba string e.g "rgba(255,0,0,1)".
    */
   onChange: PropTypes.func,
-  /**
-   * Callback fired when the popover is closed.
-   *
-   * **Signature**
-   * ```
-   * function(color: string) => void
-   * ```
-   * color: This is the selected colour in a rgba string e.g "rgba(255,0,0,1)".
-   */
-  onClose: PropTypes.func,
-  /**
-   * This determines if the popover is open on the intial
-   * render.
-   * @default false
-   */
-  open: PropTypes.bool,
-  /**
-   * This is the left positon of the popover when the anchor
-   * type is anchorPosition.
-   * @default 0
-   */
-  popoverPositionLeft: PropTypes.number,
-  /**
-   * This is the top positon of the popover when the anchor
-   * type is anchorPosition.
-   * @default 0
-   */
-  popoverPositionTop: PropTypes.number,
-  /**
-   * This is the width of the popover, this also effects the
-   * popover's children the colorpicker and controls.
-   * @default 250px
-   */
-  popoverWidth: PropTypes.string,
   /**
    * This determines if the RGB and HEX controls are shown.
```

Comment:
```suggestion
   * This determines if the RGB controls are shown.
```
We don't show a HEX input ever.